zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/SecurityTest.java

```diff
@@ -30,7 +29,6 @@
 @Timeout(time = 120)
 @Testcontainers
 @ZeebeIntegration
-@AutoCloseResources
```

Comment:
This file contains the `@AutoCloseResources` annotation, but does not mark any resource with the `@AutoCloseResource` annotation - thus no resource will be closed automatically.

🕵️ not sure if the `TestCluster` might be a candidate or the dynamic clients need some kind of closing